### PR TITLE
CO-3240 assignement automatique

### PR DESCRIPTION
**Pull request by jo-bloiz-ornot**

### CODE REVIEW

**Review by eicher31 — CHANGES_REQUESTED** [review]
Do the change of state in the write method.

**Comment by jo-bloiz-ornot** [comment]
Changes have been done

File: crm_request/models/request.py [hunk 1]
```diff
@@ -252,10 +249,18 @@ def write(self, values):
         """
         super().write(values)
 
-        if values.get("stage_id") == self.env.ref("crm_request.stage_wait_support").id:
+        # Moving a request to waiting for support
+        # automaticaly assign it to yourself
+        if values.get("stage_id") == self.env.ref(
+                "crm_request.stage_wait_support").id:
```
Comment (eicher31) [inline crm_request/models/request.py]: Also  in state resolve. please

File: crm_request/models/request.py [hunk 2]
```diff
@@ -252,10 +249,18 @@ def write(self, values):
...
             for request in self:
                 if not request.user_id:
                     request.user_id = self.env.uid
+        # Assignment modification move request on waiting for support
```
Comment (eicher31) [inline crm_request/models/request.py]: Please add space before comment

**Review by eicher31 — CHANGES_REQUESTED** [review]

File: crm_request/models/request.py (hunk 1 above)
Comment (jo-bloiz-ornot) [reply crm_request/models/request.py]: there is no ref to the state resolve, should I had it the crm_request_data.xml ?

**Comment by jo-bloiz-ornot** [comment]
I just push some fixes 
right now it should be alright

File: crm_request/__manifest__.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
     "external_dependencies": {"python": ["detectlanguage"]},
     # any module necessary for this one to work correctly
     "depends": [
+        "crm_claim",  # oca_addons/crm
```
Comment (eicher31) [inline crm_request/__manifest__.py]:
```suggestion
```

File: crm_request/models/mail_compose_message.py [hunk 4]
```diff
@@ -27,6 +27,12 @@ def send_mail(self, auto_commit=False):
                 {"partner_id": gen_mail.recipient_ids[:1].id}
             )
 
+        # Assign current user to request
```
Comment (eicher31) [inline crm_request/models/mail_compose_message.py:30]:
```suggestion
        # Assign current user to requestt
```

**Pull request merged**